web/profiles/custom/os2loop/themes/os2loop_theme/templates/content-entities/flag--os2loop-subscription-node.html.twig

```diff
@@ -0,0 +1,43 @@
+{#
+/**
+ * @file
+ * Default theme implementation for flag links.
+ *
+ * Available functions:
+ * - flagcount(flag, flaggable) gets the number of flaggings for the given flag and flaggable.
+ *
+ * Available variables:
+ * - attributes: HTML attributes for the link element.
+ * - title: The flag link title.
+ * - action: 'flag' or 'unflag'
+ * - flag: The flag object.
+ * - flaggable: The flaggable entity.
+ */
+#}
+{% spaceless %}
+{# Attach the flag CSS library.#}
+{{ attach_library('flag/flag.link') }}
+
+{# Depending on the flag action, set the appropriate action class. #}
+{% if action == 'unflag' %}
+  {% set action_class = 'unsubscribe' %}
+{% else %}
+  {% set action_class = 'subscribe' %}
+{% endif %}
```

Comment:
Should be written as `{% set action_class = action == 'unflag' ? 'unsubscribe' : 'subscribe' %}`.

Reply:
Changed